soc/xtensa/intel_adsp/Kconfig

```diff
@@ -46,7 +46,7 @@ config MEMORY_WIN_1_SIZE
 
 config MEMORY_WIN_2_SIZE
 	int "Size of memory window 2"
-	default 8192
+	default 32768
```

Comment:
Previously it was 2 pages of memory, why it is now 4 times more?

Reply:
Hi, there is one page for each slot. According to the documentation, you have to fit 7 slots into this window. There is also one page for descriptors. This gives us 8 pages in total.

Reply:
I overlooked the possibility to override this values. I moved those changes to soc/xtensa/intel_adsp/ace/Kconfig.defconfig.series. Seems to work fine. Default values can be changed when the time comes.